static/sass/_utility_crop.scss

```diff
@@ -0,0 +1,31 @@
+@mixin blog-u-crop {
+  // Crop to 16:9
+  .u-crop--16-9 {
```

Comment:
Is this the same as [embedded media](https://vanillaframework.io/docs/utilities/embedded-media)?